foreshadow/metrics/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""Custom foreshadow metrics for computing data statistics."""
+
+import glob
+import inspect
+import os
+
+
+def _get_classes():
```

Comment:
Can we create an issue to turn this into a utility, since this code is also used in transformers base.